search-api/tests/unit/services/test_validator.py

```diff
@@ -108,3 +129,44 @@ def test_document_access_request_invalid(client, session, jwt, requests_mock, te
     # check
 
     assert err[0]['error'] == error_message
+
+
+@pytest.mark.parametrize('test_name, error_message', [
+    ('no_identifier', 'Identifier is required.'),
+    ('no_legal_name', 'Business Name is required.'),
+    ('no_legal_type', 'Business Type is required.'),
+    ('no_status', 'A valid business status is required.'),
+    ('no_party_type', 'Party Type is required.'),
+    ('invalid_party_type', 'Invalid party type.'),
+    ('no_roles', 'Party Roles is required.'),
+    ('no_person_name', 'First name or middle name or last name is required.'),
+    ('no_org_name', 'Organization name is required.'),
+])
+def test_validate_solr_update_request(client, session, test_name, error_message):
+    """Assert that a auth-api user orgs request works as expected with the mock service endpoint."""
+    # setup
```

Comment:
could you put an assert at the beginning to validate that the original passes before you delete anything? (just in case someone tries to change this payload a year or two from now)

Reply:
Added